utils/src/migration.rs

```diff
@@ -1,8 +1,12 @@
-use anyhow::Result;
 use std::cmp::Ordering;
 
+use anyhow::Result;
+
 /// Compare two CRD version
-pub(crate) fn compare_versions(version1: &str, version2: &str) -> Result<Ordering> {
+/// # Errors
+/// If the version dose not match the `v<number>`
+#[inline]
+pub fn compare_versions(version1: &str, version2: &str) -> Result<Ordering> {
     let v1_parts: u32 = version1.trim_start_matches('v').parse()?;
```

Comment:
This method will return an error when using it to compare `v0.1.0` and `v0.2.0`.

Reply:
This is just a comparison of the API versions in CRD. In k8s, these versions are often written as v1, v2, v1beta and so on. Currently, we only have version v1. In the future, if there are any irregular versions such as v1beta, this function will be modified.

Reply:
Resolved in #11